<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bulkompare</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="infoLbl">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="filterLe"/>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="summaryTab">
       <attribute name="title">
        <string>Résumé</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="summaryTw">
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="differencesTab">
       <attribute name="title">
        <string>Différences</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="differencesTw">
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="inOneTab">
       <attribute name="title">
        <string>Un set seulement</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="inOneTw">
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="notComparedTab">
       <attribute name="title">
        <string>Indexes multiples</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="notComparedTw">
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QWidget" name="logoWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="logoWidgetLayout">
       <property name="topMargin">
        <number>12</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_2" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>15</width>
     <height>15</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelectDirs"/>
   <addaction name="separator"/>
   <addaction name="actionSelectProperties"/>
   <addaction name="actionSelectMapping"/>
   <addaction name="actionSelectColumns"/>
   <addaction name="separator"/>
   <addaction name="actionCompare"/>
   <addaction name="actionToggleFilter"/>
   <addaction name="actionExportExcel"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionSelectDirs">
   <property name="text">
    <string>Répertoires et extensions</string>
   </property>
  </action>
  <action name="actionSelectProperties">
   <property name="text">
    <string>Propriétés CSV</string>
   </property>
   <property name="toolTip">
    <string>Propriétés CSV</string>
   </property>
  </action>
  <action name="actionSelectMapping">
   <property name="text">
    <string>Renommer des colonnes</string>
   </property>
  </action>
  <action name="actionSelectColumns">
   <property name="text">
    <string>Choisir les colonnes</string>
   </property>
  </action>
  <action name="actionCompare">
   <property name="text">
    <string>Lancer la comparaison</string>
   </property>
  </action>
  <action name="actionToggleFilter">
   <property name="text">
    <string>Filtrage</string>
   </property>
  </action>
  <action name="actionExportExcel">
   <property name="text">
    <string>Export Excel</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
